src/configuration/app-containers.md

```diff
@@ -254,35 +255,35 @@ web:
             ...
 ```
 
-It has a few subkeys, which are:
+It has a few subkeys listed below:
```

Comment:
Hm. I think this technically would be read as "there are subkeys listed below, some of which are used by web.locations", where the actual intent is "web.locations has subkeys. An exhaustive list of them is below."